build-tools/src/main/java/org/elasticsearch/gradle/test/GradleTestPolicySetupPlugin.java

```diff
@@ -23,8 +26,13 @@ public void apply(Project project) {
             test.systemProperty("tests.gradle", true);
             test.systemProperty("tests.task", test.getPath());
 
-            // Flag is required for later Java versions since our tests use a custom security manager
-            test.jvmArgs("-Djava.security.manager=allow");
+            test.getJvmArgumentProviders().add(() -> {
```

Comment:
@mark-vieira I was wondering, do we need this? 
Isn't it already set by the ElasticsearchTestBasePlugin?

Reply:
Yes, this is needed. This is the plugin used by external plugin authors.